gravity.sh

```diff
@@ -524,15 +524,20 @@ num_target_lines=0
 num_source_lines=0
 num_invalid=0
 parseList() {
-  local adlistID="${1}" src="${2}" target="${3}" incorrect_lines
+  local adlistID="${1}" src="${2}" target="${3}" incorrect_lines sample_incorrect_lines
   # This sed does the following things:
+  # 0. Remove all lines containing no domains
```

Comment:
Can we make this a 1-based list rather than a 0-based one, please?

Reply:
Sure.